index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"/>
+
+	<title>My Portifolio Page</title>
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+	<div class="main">
+		<section class="toolbar">
+		<div class="logo">
+			<h1>My Logo</h1>
```

Comment:
Consider not using an `h1` tag here, you can make it a `p` tag or any other tag you want. `h1` tag is special and should be used once per page and is reserved for the most important heading in the page.